### PR TITLE
feat: support excel form view

### PR DESCRIPTION
**Pull request by caoxing9**

### CODE REVIEW

File: plugins/src/app/excel-form-view/components/SharePopover.tsx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+            disabled={enableShareLoading || disableShareLoading || !permission['view|share']}
+            onCheckedChange={setEnableShare}
+          />
+        </div>
+        <Separator />
+        {enableShare ? (
+          <>
+            <div className="flex items-center gap-1">
+              <Label className="sr-only" htmlFor="share-link">
+                Share Link
```
Comment (Sky-FE) [inline plugins/src/app/excel-form-view/components/SharePopover.tsx]: i18n

File: plugins/package.json [hunk 2]
```diff
@@ -21,22 +21,42 @@
     "@teable/openapi": "workspace:^",
     "@teable/sdk": "workspace:^",
     "@teable/ui-lib": "workspace:^",
+    "@types/lodash": "4.17.0",
```
Comment (boris-w) [inline plugins/package.json]: Type dependencies to dev.

File: plugins/src/app/excel-form-view/components/ExcelView.tsx [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    return (
+      <div className="flex size-full items-center justify-center">
+        <Spin />
+      </div>
+    );
+  }
+
+  return (
+    <div className="flex size-full flex-1 flex-col overflow-hidden">
+      {!isLoading && (
```
Comment (boris-w) [inline plugins/src/app/excel-form-view/components/ExcelView.tsx]: Repeat!

File: plugins/src/app/excel-form-view/components/excel/PreviewPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (univerRef?.current && fields?.length) {
+        const recordMap = getRecordsMap();
+        Object.values(recordMap).forEach(({ cellCoordinate, fieldIns }) => {
+          setCellRules(fieldIns, cellCoordinate);
+        });
+      }
+    }, 0);
```
Comment (boris-w) [inline plugins/src/app/excel-form-view/components/excel/PreviewPanel.tsx]: clear timeout

**Review by boris-w — APPROVED** [review]

**Pull request merged**